news/_posts/2017-03-06-roadmap-2.13.md

```diff
@@ -0,0 +1,28 @@
+  - [user-friendliness](https://github.com/scala/scala-dev/labels/t%3Afriendliness)
+
+We intend to shorten the development cycle a little compared to 2.12, with the first 2.13 release candidate slated for the end of Q1 2018. In 2017, we will release quarterly 2.13 [milestones](https://github.com/scala/scala/milestones) so you can get a good sense of the library changes. While Scala 2.11 will come to a halt with 2.11.9 in Q1 2017, we will maintain 2.12 with regular minor releases throughout 2017.
+
+<!-- break -->
+
+In collaboration with the Scala Center, as outlined in [SCP-007: Collaborative redesign and implementation of Scala 2.13's collections library](https://github.com/scalacenter/advisoryboard/blob/master/proposals/007-collections.md) and [Stefan's blog post](http://scala-lang.org/blog/2017/02/28/collections-rework.html), the collections library rework effort has kicked off at https://github.com/scala/collection-strawman. Our goal is to simplifying usage and improve performance, with a smooth migration path from the current collections. Please [join](https://github.com/scala/collection-strawman) and help shape one of the defining parts of Scala!
+
+In tandem, the modularisation effort of Scala 2.11 will continue as part of the [Scala Platform process](https://www.scala-lang.org/blog/2016/11/28/spp.html) to foster innovation in our eco-system while maintaining a stable core. The core consists of the collections and the other standard types, such as `Option`, `TupleN`, `Either` and `Try`.  Ultimately, we'd like provide even stronger binary compatibility guarantees for the core, with a vibrant complement of modules that evolve more quickly (offering only backwards compatibility).  [Discuss this theme]( https://github.com/scala/scala-dev/issues/323).
+
+Scala 2.13 is a library release, which means the language itself won't change. However, we continue to invest heavily in the implementation, with a single goal: the make the compiler faster. The first phase of this work involves benchmarking infrastructure, to guide our progress towards a faster compiler. After that, we will attack slowness on all fronts with all available tools. In addition to YourKit and other JVM profilers, which we have been using intensively over the years, we now have a JMH benchmarking harness for the compiler. We are also investigating lower-level performance (cpu caches, memory access patterns, JIT profiles, etc). We are eager to hear your ideas at the compiler performance theme issue. Please head over to https://github.com/scala/compiler-benchmark for the benchmarks and the issue tracker, or [discuss this theme here](https://github.com/scala/scala-dev/issues/322).
+
+Finally, there are [many more things](https://github.com/scala/scala-dev/issues?q=is%3Aopen+is%3Aissue+milestone%3A2.13+label%3Awishlist) we'd like to include in 2.13. Let us know if you'd like to work on one of these. We're always happy to help!
```

Comment:
this has to be weighed vs brevity, but maybe add "You'll find our active and friendly open-source community on [Scala Contributors](http://contributors.scala-lang.org), [scala/contributors](https://gitter.im/scala/contributors), and on GitHub issues and pull requests". the Discourse thing is still quite new and hasn't sunk in everywhere, and even the rise of Gitter isn't totally old news